doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -276,6 +276,30 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
        enable virtio poll mode with poll interval 1ms.
 
+   * - :kbd:`--acpidev_pt <HID>`
+     - This option is to enable ACPI device passthrough support. The HID is a
+       mandatory parameter for this option which is the Hardware ID of the ACPI
+       device.
+
+       Example::
+
+          --acpidev_pt MSFT0101
+
+       To pass through a TPM (which HID is MSFT0101) ACPI device to a UOS.
+
+   * - :kbd:`--mmiodev_pt <MMIO_Regions>`
+     - This option is to enable MMIO device passthrough support. The MMIO_Regions
+       is a mandatory parameter for this option which are the MMIO resources of the
+       MMIO device.
```

Comment:
Based on the example below, it sounds like that parameter needs to be the base address followed by the length of the region, both separated by a comma. Is that correct? If true, can you explain that here?

Can one include multiple regions using that parameter? If so, what separator should be used between two regions?

Reply:
Hi Geoffroy 
Thanks for your comments. Yes, will explain this. Besides, now we only support one MMIO region. Will change the MMIO_regions to MMIO_region. Is this OK. (Will detail how we separate MMIO regions when we support it.)